meeting-notes/2017-11-20-bitswap-sync.md

```diff
@@ -0,0 +1,47 @@
+# Bitswap Research Call - Nov 20, 2017
+
+## Participants
+
+@dgrisham
+@diasdavid
+
+## Agenda
+
+- Bistwap analysis
+    - https://github.com/ipfs/research-bitswap/blob/docs/strategy_analysis/analysis/prelim_strategy_analysis.pdf
+    - [x] Open a WIP PR
+        - https://github.com/ipfs/research-bitswap/pull/8
+    - [ ] Create issues for the TODO items that require thinking
+    - [ ] Present this analysis at the next IPFS All Hands
+
+
+- Bitswap metrics
+    - https://github.com/ipfs/ipfs-metrics/issues/1
+    - ipfs-performance-profiling
+    - kubernetes-ipfs
+    - ipfs/test-lab - https://github.com/ipfs/test-lab
+    - iptb
+    - [ ] Coalesce, open discussion to converge all of these efforts. Essentially create a Working Group for these things.
+
+
+- Bitswap spec
+    - First pass this week
+    - Add Bitswap sessions section
+
+
+- Bitswap Contest
+    - Learn from other academic contests
+        - How was the TFT (tit for tat) contest run?
+        - http://udt.sourceforge.net/
+        - Search for any other similar challenges
+    - What are the things we need to have to run this challenge:
+        - Interface to tinker with bitswap (the critical of the critical path) - Works start in January
+            - custom strategy input
+            - collect metrics and graphing
+        - Infrastructure (can be done in parallel)
+            - Where are these challenges running? test-lab?
+            - What is the capacity?
+        - Logistics
+            - Onsite vs offsite (hybrid)
+    - New candidate date
+        - Around Q2 or Q3 of 2018   
```

Comment:
@miyazono calling your attention to this point. We talked about the Bitswap Contest and we concluded that:

- The critical path is having at least one researched (David Grisham) be able to run through multiple strategies easily
- We need to postpone the contest to Q2 or Q3 as most of the work will only be done in Q1
- The Infrastructure work is on its way